install_config/cluster_metrics.adoc

```diff
@@ -12,307 +12,433 @@ toc::[]
+link:../install_config/cluster_metrics.html#deployer-template-parameters[template parameters]
+when calling these commands.
+
+.Deploying with Persistent Storage
 ====
-[source,yaml]
+The following command sets the Hawkular Metrics route to use
+`hawkular-metrics.example.com` and is deployed using persistent storage.
+
+You must have a persistent volume of sufficient size available.
+
+[options="nowrap"]
 ----
-apiVersion: "v1"
-kind: "List"
-items:
-  -
-    apiVersion: "v1"
-    kind: "ReplicationController"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "heapster"
-      name: "heapster"
-    spec:
-      replicas: 1
-      selector:
-        provider: "fabric8"
-        component: "heapster"
-      template:
-        metadata:
-          labels:
-            provider: "fabric8"
-            component: "heapster"
-        spec:
-          containers:
-            -
-              args:
-                - "-source=kubernetes:https://kubernetes.default.svc?auth=&insecure=true&useServiceAccount=true"
-                - "-sink=influxdb:http://influxdb-monitoring:8086"
-              image: "kubernetes/heapster:v0.17.0"
-              name: "heapster"
-          serviceAccount: "heapster"
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,IMAGE_PREFIX=openshift/origin-,IMAGE_VERSION=devel \
+| oc create -f -
 ----
 ====
 
-Save it to a file and create it with:
+.Deploying without Persistent Storage
+====
+The following command sets the Hawkular Metrics route to use
+`hawkular-metrics.example.com` and is deployed without persistent storage.
+Remember, this is being deployed without persistent storage, so metric data loss
+can occur.
 
+[options="nowrap"]
 ----
-$ oc create -f <path/to/heapster.yaml>
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,IMAGE_PREFIX=openshift/origin-,IMAGE_VERSION=devel,USE_PERSISTENT_STORAGE=false \
+| oc create -f -
 ----
+====
 
-== Deploying Grafana
+== Configuring OpenShift
 
-*Grafana* allows users to create dashboards of metrics from *InfluxDB*. The
-default installation comes with a basic dashboard, and users are encouraged to
-create their own.
+The OpenShift web console uses the data coming from the Hawkular Metrics service
+to display its graphs. The URL for accessing the Hawkular Metrics service
+must be configured via the `*metricsPublicURL*` option in the
+link:../install_config/master_node_configuration.html#master-configuration-files[master-config.yaml]
+file. This URL corresponds to the route created with the
+`*HAWKULAR_METRICS_HOSTNAME*` template parameter during the
+link:../install_config/cluster_metrics.html#deploying-the-metric-components[deployment]
+of the metrics components.
 
-The following is the definition of the *Grafana* service and replication
-controller:
+[NOTE]
+====
+You must be able to resolve the `*HAWKULAR_METRICS_HOSTNAME*` from the browser
+accessing the console.
```

Comment:
I'd move this note above to where you describe `HAWKULAR_METRICS_HOSTNAME` above. If you already created the route, it's too late now...